src/coreclr/tryrun.cmake

```diff
@@ -82,9 +82,7 @@ if(TARGET_ARCH_NAME MATCHES "^(armel|arm|arm64|x86)$" OR FREEBSD OR ILLUMOS)
     set_cache_value(HAVE_COMPATIBLE_LOG10_EXITCODE 1)
     set_cache_value(HAVE_COMPATIBLE_LOG_EXITCODE 1)
     set_cache_value(HAVE_LARGE_SNPRINTF_SUPPORT_EXITCODE 1)
-    set_cache_value(HAVE_PROCFS_CTL_EXITCODE 1)
-    set_cache_value(HAVE_PROCFS_STATUS_EXITCODE 1)
-    set_cache_value(HAVE_CLOCK_THREAD_CPUTIME_EXITCODE 1)
+    set_cache_value(HAVE_PROCFS_CTL_EXITCODE 0)
```

Comment:
@janvorli, this was the leftover from #37753. The `_EXITCODE` conditions needed to be inverted. Removed `HAVE_PROCFS_STATUS_EXITCODE ` and `HAVE_CLOCK_THREAD_CPUTIME_EXITCODE` because they are already set to 0 above.